src/Util/ColumnParser.php

```diff
@@ -178,6 +191,30 @@ public function getTypeAndLength($field, $type)
     }
 
     /**
+     * Get the defaults of a field based on the field passed
+     *
+     * @param string $field Name of field
+     * @return array First value is the default value for the field, second value is the if field is null. If no length
+     * can be extracted, null is returned for default value and false for null
+     */
+    public function getDefaults($field)
+    {
+
+        if (preg_match($this->regexpParseFieldDefaults, $field, $matches)) {
+            $options = explode(':', trim($matches[0], '[]'));
+            if (isset($options[1])) {
+                if ($options[1] == 'null') $matches['null'] = true;
```

Comment:
Inline ``if`` is not allowed as per [our coding standards](http://book.cakephp.org/3.0/en/contributing/cakephp-coding-conventions.html)